src/styles/blocks/stars--2.css

```diff
@@ -0,0 +1,44 @@
+.stars--2 {
+  display: flex;
+  margin-top: 4px;
```

Comment:
Be consistent with your margins (Add only top or bottom, but not both). Here you have a margin-top, ensure that you are not adding margin-bottom in related elements to avoid potential margin collapse.